src/composables/useEnv.ts

```diff
@@ -0,0 +1,20 @@
+type EnvVars = {
+  KUMA_VERSION: string
+  KUMA_DOCS_URL: string
+  KUMA_UTM_QUERY_PARAMS: string
+  KUMA_FEEDBACK_URL: string
+}
+export const useEnv = ({ version = '' }) => {
```

Comment:
I think I originally passed it in from where the value already was to avoid changing things too much. Probably won't need to pass a version from the `store` in any more once we can get that value from elsewhere, so this will change slightly once other code comes up here.